capacity4more/modules/c4m/og/c4m_og/c4m_og.module

```diff
@@ -337,6 +337,7 @@ function c4m_og_node_access_records_alter(&$grants, $node) {
  *  Entity IDs of the public groups.
  */
 function c4m_og_get_groups_by_access_type($access) {
```

Comment:
The title of this function is misleading: is suggests that you can get all groups by their access type (public, restricted, private).

Thats not the case.

Why should we pass the public 0/1 parameter, can't we rely on the node_access system to get only the groups the user has access to?